frontend/src/features/mapSideBar/SidebarContents/BuildingForm.tsx

```diff
@@ -36,35 +53,120 @@ const FillRemainingSpace = styled.span`
   flex: 1 1 auto;
 `;
 
-const Form = () => {
+interface IFormProps {
```

Comment:
We need to document these types of properties moving forward.  Makes it much harder for a new dev looking at this component to know what they all mean and do.

Reply:
One of my tasks today I have planned for today is to through and document, but I should start doing it as I create the forms as better practice.  Will start incorporating that into my process!